onnxruntime/contrib_ops/cuda/transformers/generation_device_helper.cc

```diff
@@ -1073,7 +1072,7 @@ Status UpdateDecoderFeeds(
                                          cudaMemcpyHostToDevice, cuda_stream));
   } else {
     int32_t* input_ids_data = input_ids.GetMutable<Tensor>()->MutableData<int32_t>();
-    for (int i = 0; i < batch_beam_size; i++) {
+    for (size_t i = 0; i < batch_beam_size; i++) {
```

Comment:
int64_t is better here. May need explicit cast to int in the next line.

Reply:
Reverted the change.